tests/Functional/Execution/SchemaTest.php

```diff
@@ -279,9 +279,9 @@ public function testExecuteUsingASchemaWithCustomScalarType()
         $blogArticle = newObjectType([
             'name'   => 'Article',
             'fields' => [
-                'id'        => ['type' => newNonNull(String())],
-                'title'     => ['type' => String()],
-                'body'      => ['type' => String()],
+                'id'        => ['type' => newNonNull(StringType())],
```

Comment:
@crisu83 Why don't we also have `NewNonNull()` function? Why capitalize `StringType()` but not `NewNonNull()`? Just a small thing but it make my eyes itchy, because I think it's public API and it's not consistency. Is there any change that we change it later 😵

Reply:
You're correct, I'll change these types to be non-capitalized.

Reply:
Okay 🎖